src/DrTests/DrTests.class.st

```diff
@@ -377,36 +383,47 @@ DrTests >> switchUI [
 
 { #category : #accessing }
 DrTests >> testsConfiguration [
-	^ DTPluginConfiguration
-		items: self itemsSelected
-		packages: self packagesSelected
+	^ self currentPlugin buildConfigurationFrom: self
 ]
 
 { #category : #accessing }
 DrTests >> testsConfiguration: aDTTestsConfiguration [
 	self packagesList model shownItems
 		withIndexDo: [ :package :i | 
 			(aDTTestsConfiguration packagesSelected
-				anySatisfy: [ :p | p name = package name ])
+				anySatisfy: [ :p | p = package ])
```

Comment:
are `#=` and `#hash` defined on RPackage now?

Reply:
yeah! now they work fine